marty/js/stores/CartStore.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var Marty = require('marty');
+var assign = require('object-assign');
+var CartConstants = require('../constants/cartConstants');
+
+var CartStore = Marty.createStore({
+  displayName: 'Cart',
+  handlers: {
+    onAddToCart: CartConstants.ADD_TO_CART,
+    onCartCheckout: CartConstants.CART_CHECKOUT,
+    onSuccessCheckout: CartConstants.SUCCESS_CHECKOUT
+  },
+  getInitialState: function () {
+    return {}
+  },
+  onAddToCart: function (product) {
+    var id = product.id;
```

Comment:
@jhollingworth CartStore should wait for ProductStore in order to proceed with data manipulation https://github.com/voronianski/flux-comparison/blob/master/facebook-flux/js/stores/CartStore.js#L55